.github/workflows/test-and-ship.yml

```diff
@@ -70,13 +70,77 @@ jobs:
     timeout-minutes: 15
     continue-on-error: true
 
+    services:
+      mysql:
+        image: mysql:5.7
+        ports:
+          - "33061:3306"
+        env:
+          MYSQL_ROOT_PASSWORD: root
```

Comment:
@tuxpiper are these okay?

Reply:
yes it's ok @AmTryingMyBest  . This is an ephemeral, non-publicly reachable mysql service